test/storage_snapper_configure_snapper_test.rb

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env rspec
+
+ENV["Y2DIR"] = File.expand_path("../../src", __FILE__)
+
+require "yast"
+
+Yast.import "StorageSnapper"
+Yast.import "Storage"
+
+
+describe "StorageSnapper#configure_snapper?" do
```

Comment:
This syntax for stubbing is deprecated and this still not address the original concern of poor output. Compare this:
```
StorageSnapper#configure_snapper?
  configures snapper
  does not configure snapper
  does not configure snapper
  does not configure snapper

Deprecation Warnings:

Using `stub` from rspec-mocks' old `:should` syntax without explicitly enabling the syntax is deprecated.
Use the new `:expect` syntax or explicitly enable `:should` instead.
Called from /.../storage_snapper_configure_snapper_test.rb:23:in `block (2 levels) in <top (required)>'.
```
to this (just an example, it can be done with less nesting as well):
```
StorageSnapper#configure_snapper?
  when root is btrfs
    when snapshots are enabled
      it returns true
    when snapshots are disabled
      it returns false
  when root is xfs
    when snapshots are enabled
      it returns false
    when snapshots are disabled
      it returns false
```
Anyways, even if we don't agree in the "test output as readable documentation" approach, please fix the deprecation warnings (see below).